generators/server/templates/src/main/java/package/web/rest/UserResource.java.ejs

```diff
@@ -149,6 +153,7 @@ public class UserResource {
         <%_ if (authenticationType !== 'oauth2') { _%>
         this.userRepository = userRepository;
         this.mailService = mailService;
+        this.forbiddenOrderedProperties = Collections.unmodifiableList(List.of("resetKey", "password"));
```

Comment:
Make it a constant ?

Reply:
Should we add `activationKey`, `resetDate` and `persistentTokens` (for session, guess we remove it in 7.x) too?

Reply:
What do you think @laxaa ?

Reply:
I would have advised to remove the unnecessary fields from the model if its possible.
Otherwise, I would rather go for a white-list rather than a black-list, might avoid future problems that way if your model evolves.

Reply:
hmmm, it's a great idea, I'm not sure we can remove fields at the moment. Let's do a white list so.